src/modules/helpchan.ts

```diff
@@ -33,16 +33,15 @@ export class HelpChanModule extends Module {
 	CHANNEL_PREFIX = 'help-';
 
 	AVAILABLE_EMBED = new MessageEmbed()
-		.setColor(TS_BLUE)
+		.setColor('#77b155')
```

Comment:
Add a const for this color [here](https://github.com/typescript-community/community-bot/blob/master/src/env.ts#L35)?

Reply:
Done